seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/source/reader/DorisSourceReader.java

```diff
@@ -71,7 +75,12 @@ public void pollNext(Collector<SeaTunnelRow> output) throws Exception {
             DorisSourceSplit nextSplit = splitsQueue.poll();
             if (nextSplit != null) {
                 PartitionDefinition partition = nextSplit.getPartitionDefinition();
-                valueReader = new DorisValueReader(partition, dorisConfig, seaTunnelRowType);
+                DorisSourceTable dorisSourceTable =
+                        tables.get(TablePath.of(partition.getDatabase(), partition.getTable()));
+                if (dorisSourceTable == null) {
+                    return;
```

Comment:
I think we should throw exception.

Reply:
> I think we should throw exception.
Do we need to terminate directly here? Or do we add the warn log output, what do you think?

Reply:
An exception should be thrown because if there is no bug with the connector, it should never be null

Reply:
> An exception should be thrown because if there is no bug with the connector, it should never be null

Ok, I see what you mean. I'll adjust. thx